packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -252,12 +252,16 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
     );
   }
 
-  private _onMouseUpOrTouchEnd = (): void => {
+  private _onMouseUpOrTouchEnd = (event: MouseEvent | TouchEvent): void => {
```

Comment:
Does this work without modifying lines 167 and 170? Maybe i just don't know how that _events.on callback takes in props.

Last thing then, can you add a quick test to make sure this onChanged returns the event and correct number?

Reply:
I tried to add a test previously but I didn't succeed to simulate a mousedown and then a mouseup to triggered the onChanged. 
I followed this issue https://github.com/airbnb/enzyme/issues/426 and this one : https://github.com/airbnb/enzyme/issues/1357 saying that we should not use Enzyme simulate anymore.
Could you give me some pointers?
Thanks

Reply:
For now, i'd just piggyback off of the existing onChange test and make sure your onChanged callback is being fired as well (including event). I agree, it seems enzyme is moving away from simulate, so no need to do anything elaborate.

Reply:
It can't work since onChanged is fired only when the mouseup or touchend happens.
On mousedown event (like in the extisting onChange test), onChanged won't be triggered.
I try to hack something like in the callout.test.tsx but it doesn't work : 
```
const mouseEvent = document.createEvent('MouseEvent');
    const root = document.createElement('div');
    document.body.appendChild(root);

    ReactDOM.render<HTMLDivElement>(<Slider onChanged={onChanged} />, root);

    mouseEvent.initMouseEvent('mousedown', false, false, window, 0, 0, 0, 0, 0, false, false, false, false, 1, root);
    mouseEvent.initMouseEvent('mouseup', false, false, window, 0, 0, 0, 0, 0, false, false, false, false, 1, root);
```
Thank you for your time but I'm afraid I won't be able to complete this PR :/